keyboards/butterstick/butterstick.c

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2019 %YOUR_NAME%
```

Comment:
Please put your name here

Reply:
Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0